sale_order_line_description/models/sale_order_line.py

```diff
@@ -7,9 +7,9 @@
 class SaleOrderLine(models.Model):
     _inherit = "sale.order.line"
 
-    @api.onchange("product_id")
-    def product_id_change(self):
-        res = super(SaleOrderLine, self).product_id_change()
+    @api.depends("product_id")
```

Comment:
You can remove the depends as already defined in sale module.

Reply:
done, thanks!